src/sentry/services/http.py

```diff
@@ -50,19 +42,64 @@ def __init__(self, host=None, port=None, debug=False, workers=None,
         self.workers = workers
 
         options = (settings.SENTRY_WEB_OPTIONS or {}).copy()
-        options.setdefault('bind', '%s:%s' % (self.host, self.port))
-        options.setdefault('timeout', 30)
-        options.setdefault('proc_name', 'Sentry')
+        options.setdefault('module', 'sentry.wsgi:application')
+        options.setdefault('protocol', 'http')
+        options.setdefault('http-socket', '%s:%s' % (self.host, self.port))
+        options.setdefault('auto-procname', True)
+        options.setdefault('procname-prefix-spaced', '[Sentry]')
         options.setdefault('workers', 3)
-        options.setdefault('accesslog', '-')
-        options.setdefault('errorlog', '-')
-        options.setdefault('loglevel', 'info')
-        options.setdefault('limit_request_line', 0)
-        options['preload'] = False
+        options.setdefault('threads', 1)
```

Comment:
I wanted to keep it to match existing behavior.

Should `-w` on CLI then just manage `threads`? So 1 proc + `n` threads?